.env.test

```
@@ -0,0 +1,4 @@
+NODE_ENV=test
+DATABASE_URL=DATABASE UNAVAILABLE
+GREEN_BITE_API_URL=https://green-bite.olillin.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GREEN_BITE_API_URL key should go before the NODE_ENV key

(UnorderedKey)